src/components/MainLayout/MainHeader.tsx

```diff
@@ -385,7 +387,7 @@ const CircleNav = () => {
 
   const links: [string, string, string[]?][] = useMemo(() => {
     const l: [string, string, string[]?][] = [
-      [paths.history, 'History'],
+      // [paths.contributions, 'Contributions'],
```

Comment:
should remove this line if this isn't needed anymore, or uncomment if it is

Reply:
removed